### PR TITLE
🎉 Added document for locating files in local destination on windows

### LINKED ISSUE #2213: Figure out how to properly mount docker volumes on Windows
## Tell us about the problem you're trying to solve
As an Airbyte user running on a Windows machine, I would like to be able to use local destinations/sources connectors such as LocalJson or CSV.

When syncs are run with these connectors, they store data in `/local/` in the docker volume data.
This is mounted from the host machine from `/tmp/airbyte_data` by default but this location doesn't exist on a windows machine.

## Describe the solution you’d like
Give proper instructions on how to mount a windows directory with docker, especially when using airbyte?

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368215699782) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by MohamadHaziq**
## What
Closes https://github.com/airbytehq/airbyte/issues/2213

Added a document to help users find their data when using local destination as destination on Windows and WSL2

### CODE REVIEW

File: docs/tutorials/locating-files-local-destination.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## Locating where your temp folder is
+
+While running Airbyte's Docker image on Windows with WSL2, you can access your temp folder by going to 
+
+1. Open File Explorer (Or any folder where you can access the address bar)
+2. Type in `\\wsl$` in the address bar
+3. The folders below will be displayed
+![](../.gitbook/assets/windows-wsl2-docker-folders.png)
+4. You can start digging here, but it is recommended to start searching from here and just search for the folder name you used for your local files. The folder address should be similar to `\\wsl$\docker-desktop\tmp\docker-desktop-root\containers\services\docker\rootfs\tmp\airbyte_local`
+5. You should be able to local your local destination CSV or JSON files in this folder.
```
Comment (ChristopheDuong) [inline docs/tutorials/locating-files-local-destination.md:18]:
```suggestion
5. You should be able to locate your local destination CSV or JSON files in this folder.
```

**Comment by ChristopheDuong** [comment]
Thanks for your contribution!!

I think this would solve the problem users encounters as described here https://github.com/airbytehq/airbyte/issues/2213

File: docs/tutorials/locating-files-local-destination.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Overview
+
+This tutorial will describe how to look for json and csv files in when using local destinations on Windows on a local deployment.
+
+There is generally some confusion when using local destination as your destination in Airbyte on Windows, especially if you are running WSL2 to power Docker. There are also two folders generated at the root folder of your Docker folder which will point you in the wrong direction
+
+## Locating where your temp folder is
+
+While running Airbyte's Docker image on Windows with WSL2, you can access your temp folder by going to
```
Comment (avaidyanatha) [inline docs/tutorials/locating-files-local-destination.md:11]:
```suggestion
While running Airbyte's Docker image on Windows with WSL2, you can access your temp folder by doing the following: 
```

File: docs/tutorials/locating-files-local-destination.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+1. Open File Explorer (Or any folder where you can access the address bar)
+2. Type in `\\wsl$` in the address bar
+3. The folders below will be displayed
+![](../.gitbook/assets/windows-wsl2-docker-folders.png)
+4. You can start digging here, but it is recommended to start searching from here and just search for the folder name you used for your local files. The folder address should be similar to `\\wsl$\docker-desktop\tmp\docker-desktop-root\containers\services\docker\rootfs\tmp\airbyte_local`
+5. You should be able to local your local destination CSV or JSON files in this folder.
+
+## Notes
+
+1. Local JSON and Local CSV files do not persist between docker restarts. This means once you turn off your docker image, your data is lost. This is consistent with the `tmp` nature of the folder
```
Comment (avaidyanatha) [inline docs/tutorials/locating-files-local-destination.md:22]:
```suggestion
1. Local JSON and Local CSV files do not persist between Docker restarts. This means that once you turn off your Docker image, your data is lost. This is consistent with the `tmp` nature of the folder.
```

**Review by avaidyanatha — APPROVED** [review]
Looks good, thanks for adding this!! Just some small grammar/clarity fixes.

**Comment by marcosmarxm** [comment]
Thanks @MohamadHaziq hope to see the Facebook Pages soon :rocket:

**Pull request merged**